bitnami/mariadb/templates/secrets.yaml

```diff
@@ -20,13 +20,17 @@ data:
   {{- else }}
   mariadb-root-password: {{ required "A MariaDB Root Password is required!" .Values.auth.rootPassword }}
   {{- end }}
-  {{- if and (not (empty .Values.auth.username)) (not (empty .Values.auth.password)) }}
+  mariadb-database: {{ .Values.auth.database | b64enc | quote }}
+  {{- if not (empty .Values.auth.username) }}
+  mariadb-username: {{ .Values.auth.username | b64enc | quote }}
+  {{- if not (empty .Values.auth.password) }}
```

Comment:
This is something we have discussed in the past, and we decided only to include passwords in the secret, not considering the rest of authentication parameters as "sensitive data".
The `auth.database` and `auth.username` are set directly in MariaDB/MySQL statefulset(s).

Reply:
@juan131 I understand your position. But it's not handy to search this fields in statefulset for further referencing. I can move these fields in configmap. Should I ?